R/R_rap_demo/cleaning.R

```diff
@@ -0,0 +1,24 @@
+#cleaning.R: Data cleaning functions for RAP pipeline
+
+clean_health_data <- function(df) {
+  
+  #Clean health data by handling missing values and standardizing columns.
+
+  #Args:
+  #    df (DataFrame): Raw health data DataFrame.
+  
+  #Returns:
+  #    df_clean (DataFrame): Cleaned health data DataFrame.
```

Comment:
For all of the functions, I think we should use [Roxygen](https://cran.r-project.org/web/packages/roxygen2/vignettes/roxygen2.html) documentation 

They look like the below:
```suggestion
  #' Clean health data by handling missing values and standardizing columns.
  #' 
  #' @param df raw health data DataFrame.
  #'
  #' @return df_clean cleaned health data DataFrame.
```
You can then create manuals automatically using roxygen2::roxygenise()